Sanity/pkcs11/basic/runtest.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/clevis/Sanity/basic/pkcs11
+#   Description: tests the basic pkcs11 functionality of clevis
+#   Author: Martin Litwora <mlitwora@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2024 Red Hat, Inc.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || :
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+
+create_hsm_config() {
+    # Create a configuration file
+    cat > $TMPDIR/softhsm.conf << EOF
+directories.tokendir = $TMPDIR
+objectstore.backend = file
+log.level = DEBUG
+EOF
+}
+
+install_softhsm() {
```

Comment:
You can add adjust to **main.fmf** to restrict executing this test just for specific distros.

Reply:
There is already the:
```
adjust:
  - enabled: false
    when: distro == rhel-4, rhel-5, rhel-6, rhel-7, rhel-8
```
in the main.fmf file